server/chat.ts

```diff
@@ -219,7 +219,6 @@ export abstract class MessageContext {
 		}
 		return false;
 	}
-
 	tr(strings: TemplateStringsArray | string, ...keys: any[]) {
```

Comment:
```suggestion

	tr(strings: TemplateStringsArray | string, ...keys: any[]) {
```
(you seem to have accidentally deleted a linebreak)